src/index.js

```diff
@@ -30,15 +30,13 @@ setInterval(() => {
   if (data.length > 0) {
     data.forEach((myItem) => {
       const parsedTitle = getTagsFromTitle(myItem.Title);
-      g.set({
+      const labels = {
         customer: parsedTitle[0], service: parsedTitle[2], device: parsedTitle[4], version: parsedTitle[6],
-      }, myItem.LatestStats.loadTimeMS, date);
-      h.set({
-        customer: parsedTitle[0], service: parsedTitle[2], device: parsedTitle[4], version: parsedTitle[6],
-      }, myItem.LatestStats.fileSizeKB, date);
-      i.set({
-        customer: parsedTitle[0], service: parsedTitle[2], device: parsedTitle[4], version: parsedTitle[6],
-      }, myItem.LatestStats.Requests, date);
+      };
+      const setter = gauge => gauge.set(labels, myItem.LatestStats.loadTimeMS, date);
```

Comment:
🙅‍♂ 
You will set the value `loadTimeMS` for each Gauge here 💣

Reply:
😱